src/main/scala/mesosphere/marathon/state/PortAssignment.scala

```diff
@@ -1,5 +1,13 @@
 package mesosphere.marathon.state
 
+/**
+  * @param portName name of the port
+  * @param portIndex index of the port, used for example to define health checks
+  * @param effectiveIpAddress ip address on which the port can be reached (can be an agent's IP or an IP-per-Task)
+  * @param effectivePort resolved non-dynamic port. The task is reachable under effectiveIpAddress:effectivePort.
```

Comment:
The port which is reachable from the outside?? 

I like "The task is reachable under effectiveIpAddress:effectivePort".